index.bs

```diff
@@ -0,0 +1,20 @@
+<pre class='metadata'>
+Title: Local Peer-to-Peer API
+Shortname: local-peer-to-peer
+Level: 1
+Status: CG-DRAFT
+Group: WICG
+Repository: WICG/local-peer-to-peer
+URL: http://example.com/url-this-spec-will-live-at
```

Comment:
```suggestion
URL: https://wicg.github.io/local-peer-to-peer/
```

This URL is where `index.html` will be published by the workflow. This metadata is used to generate the header boilerplate for that.